config/clusters/bican/common.values.yaml

```diff
@@ -0,0 +1,183 @@
+nfs:
+  enabled: true
+  pv:
+    enabled: true
+    # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+    mountOptions:
+      - rsize=1048576
+      - wsize=1048576
+      - timeo=600
+      - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+      - retrans=2
+      - noresvport
+    serverIP: fs-0f48aeaf92202d25f.efs.us-east-2.amazonaws.com
+    baseShareName: /
+jupyterhub:
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    jupyterhubConfigurator:
+      enabled: false
+    homepage:
+      templateVars:
+        org:
+          name: BICAN
+          logo_url: https://static.wixstatic.com/media/c57c96_a9e46e43008349c8b65dcacc3ceaba35~mv2.png/v1/fill/w_402,h_150,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/_BICAN-full-logo-final.png
+          url: https://www.portal.brain-bican.org/
+        designed_by:
+          name: "2i2c"
+          url: https://2i2c.org
+        operated_by:
+          name: "2i2c"
+          url: https://2i2c.org
+        funded_by:
+          name: BICAN
+          url: https://www.portal.brain-bican.org/
+
+  hub:
+    allowNamedServers: true
+    config:
+      JupyterHub:
+        authenticator_class: github
+      Authenticator:
+        admin_users:
+          - kabilar
+          - aaronkanzer
+          - asmacdo
+          - satra
+          - djarecka
+  singleuser:
+    profileList:
+      - display_name: "DANDI (CPU)"
```

Comment:
Not sure, as the tracking issue also says `dandi` but shouldn't this be Bican instead of Dandi??

Reply:
@GeorgianaElena these are DANDI images, so I think we can call them that for now. I think this will be modified in the future.